lib/membrane_h264_format/h26x/types.ex

```diff
@@ -0,0 +1,52 @@
+defmodule Membrane.H26x.Types do
+  @typedoc """
+  Width of single frame in pixels.
+
+  Allowed values may be restricted by used encoding parameters, for example, when using
+  4:2:0 chroma subsampling dimensions must be divisible by 2. If the information about the
+  width is not present in the stream, `nil` value should be used.
+  """
+  @type width :: pos_integer()
```

Comment:
I know you have moved it from the `h264,.ex` but since we are already touching it, perhaps it would be good to allow for the `nil` value:
```suggestion
  @type width :: pos_integer() | nil
```

Reply:
It also applies for some other types defined here